### ISSUE TITLE
Remove comment symbol at the end of file

### ISSUE THREAD

**Issue by dvishal485** [dvishal485/leetcode-runner-cli#7]
# Current Behavior

Using default code provided leave a comment symbol behind which seems to be appearing in the Leetcode submissions.

![image](https://github.com/dvishal485/leetcode-runner-cli/assets/26341736/f981e0ae-d9cb-4435-8a88-be213eb08883)

# Expected Behavior

The comment symbol (if any) should be removed as well from the end of file given that the comment doesn't include anything user would have wrote themselves.
Basically, remove the entire comment if it is followed by the closing delimiter `#LCEND` and trim the response before submitting.
